### ISSUE TITLE
Behavior differs by web3 version

### ISSUE THREAD

**Issue by Fang-** [urbit/azimuth-js#22]
The docs for, say, `getTransferringFor()`, describe:

```
azimuth~getTransferringFor ⇒ Promise.<Array.<Number>>
(...)
Returns: Promise.<Array.<Number>> - The points.
```

However, when I use that function, I get back an array of number strings (ie, `'256'`) instead. I bet the same happens for all other functions that return points. None have a wrapper around them that parses the strings into numbers.

**Comment by Fang-** [comment]
This depends on the web3 version that's being passed in/used. azimuth-js cannot be agnostic to this. Numbers (instead of strings) are the correct thing to do, and latest web3 does that. Pretty sure that won't change in the future, so I think we should just say "use a modern web3 version if you want the docs/complex behavior to be correct".

I pushed some WIP to the `new-web3` branch. It gets some of the way there to making tests succeed with latest-ish web3, but some transactions are still being rejected for unknown reasons. We'll have to return to that at some point.

**Comment by jtobin** [comment]
On Fri, Jul 19, 2019 at 10:39:02AM -0700, Fang wrote:
> This depends on the web3 version that's being passed in/used.
> azimuth-js cannot be agnostic to this. Numbers (instead of strings)
> are the correct thing to do, and latest web3 does that.

The 'peerDependencies' of package.json specifies the appropriate web3
versions that azimuth-js supports; it's currently at >= beta.33, so we
should bump that to >= beta.55 or whatever alongside changes that make
that the case.

**Comment by jtobin** [comment]
Resolved in #46.
